<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>665</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>梦魇Wave</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>641</width>
        <height>481</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QTableWidget" name="source_tw">
             <attribute name="horizontalHeaderDefaultSectionSize">
              <number>300</number>
             </attribute>
             <attribute name="horizontalHeaderMinimumSectionSize">
              <number>1</number>
             </attribute>
             <attribute name="verticalHeaderDefaultSectionSize">
              <number>30</number>
             </attribute>
             <column>
              <property name="text">
               <string>ctrl</string>
              </property>
             </column>
            </widget>
           </item>
           <item>
            <widget class="QTableWidget" name="target_tw">
             <attribute name="horizontalHeaderDefaultSectionSize">
              <number>300</number>
             </attribute>
             <attribute name="verticalHeaderDefaultSectionSize">
              <number>30</number>
             </attribute>
             <column>
              <property name="text">
               <string>controlled</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>总控制器：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="main_ctrl">
             <property name="maximumSize">
              <size>
               <width>202</width>
               <height>202</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="input_main_ctrl_pb">
             <property name="text">
              <string>《《</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_8">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Expanding</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="doit_pb">
             <property name="text">
              <string>执行</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>控制器：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="load_source_pb">
           <property name="text">
            <string>load↓</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="source_clear_pb">
           <property name="text">
            <string>clear</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>被控制体：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="load_target_pb">
           <property name="text">
            <string>load↓</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="target_clear_pb">
           <property name="text">
            <string>clear</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="Line" name="line_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>500</y>
        <width>637</width>
        <height>3</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="add_offset_doit_pb">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>510</y>
        <width>111</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>增加offset属性</string>
      </property>
     </widget>
     <widget class="QPushButton" name="add_blink_pb">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>550</y>
        <width>111</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>增加眼睛随机眨动</string>
      </property>
     </widget>
     <widget class="Line" name="line_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>540</y>
        <width>637</width>
        <height>3</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>510</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>选中控制器：</string>
      </property>
     </widget>
     <widget class="QPushButton" name="delete_old_attr_pb">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>550</y>
        <width>131</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>删除原有属性（临时）</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
